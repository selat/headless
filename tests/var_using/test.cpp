#include "using_directive.hpp"

void test() {
  SuperType type;
  (void)type;
}
