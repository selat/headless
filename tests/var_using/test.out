Please add the following headers:
Please remove the following headers:
